Decred Wallet/Features/Wallet Utils/Syncer.swift

```diff
@@ -214,6 +214,13 @@ extension Syncer: DcrlibwalletSyncProgressListenerProtocol {
         self.currentSyncOp = .Done
         self.currentSyncOpProgress = nil
         self.forEachSyncListener({ syncListener in syncListener.onSyncCompleted() })
+
+        let isNewWalletSetup: Bool = Settings.readValue(for: Settings.Keys.NewWalletSetUp)
+        if let initialSyncCompleted: Bool = Settings.readOptionalValue(for: Settings.Keys.InitialSyncCompleted) {
```

Comment:
Regardless of whether this is a new wallet or a restored wallet, it is not out of place to set the `InitialSyncCompleted = true` value.
If we must check anything, we can prolly check if the value wasn't already set before setting it.